ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Node.hs

```diff
@@ -284,4 +287,5 @@ instance NodeInitStorage ByronBlock where
 instance BlockSupportsMetrics ByronBlock where
   isSelfIssued = isSelfIssuedConstUnknown
 
-instance RunNode ByronBlock
+instance SerialiseNodeToClient ByronBlock (LedgerConfig ByronBlock)
```

Comment:
Doesn't the `SerialiseNodeToClient ByronBlock Cardano.Chain.Genesis.Config` instance in the ` Ouroboros.Consensus.Byron.Node.Serialisation` module provide this? Can we drop this new instance context element?

Reply:
Yes! Will delete